sktime/pipeline/pipeline.py

```diff
@@ -157,6 +158,13 @@ class Pipeline(BaseEstimator):
     """
 
     def __init__(self, steps=None):
+        warnings.warn(
+            "This generalised graphical pipeline is experimental. "
+            "Thus, there are usual risk with cutting edge features. "
+            "If you are searching in a mature feature, please use the "
+            "the existing sequential pipeline, e.g. ForecastingPipeline.",
```

Comment:
some thoughts on this:

* perhaps point people to an issue?
* "if you are serching in" -> odd English
* perhaps point people to the classes but also the pipelines notebook?